SConstruct

```diff
@@ -19,6 +19,12 @@ DESIRED_FLAGS = ['-Wno-gnu-designator',
 PP_ONLY_FLAGS = ['-Wno-parentheses-equality', '-Wno-builtin-requires-header',
                  '-Wno-unused-function']
 
+def need_scons_local():
+    """Print scons_local warning and exit"""
+    print ('scons_local submodule is needed in order to do DAOS build')
+    print ('Use git submodule init; git submodule update')
```

Comment:
Really just informational and I'm not sure if this operation as expressed in two separate `git submodule` commands will ever go away but I have observed:

```suggestion
    print ('Use git submodule update --init')
```

as an alternative.

Again, mostly just FYI.

But more to the point, is there any reason we wouldn't just do the `git submodule init; git submodule update` here for the user, or is the horse out of the barn at this point?

Reply:
I didn't know that was an option so thanks for pointing it out.   I don't think we should do it automatically for them.   If, for example, I'm doing local experiments on something (maybe another submodule), I might be annoyed if a build changed the state